client/src/components/Login.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react"
+import {ThemeProvider, theme, CSSReset, Flex, Box, Heading, Link, FormControl, FormLabel, Input, Stack, Checkbox, Button} from "@chakra-ui/react"
+
+const Login = () => {
```

Comment:
Can you put the return inside a render() method and convert the Login function into a class? Here's an example. 
![image](https://user-images.githubusercontent.com/24682960/109920985-0570a700-7cf6-11eb-9049-8c7f68756312.png)

You should add a function under the class that allows the log in form to be submitted (for example you can call it `onSubmitLogin()`), and then you should pass this function as a prop to LoginForm, and call the function on click of the submit button. Remember to do the same thing for the go to registration button.

For more info look at main concepts 4 and 5 from the react docs: https://reactjs.org/docs/state-and-lifecycle.html